docs/source/configuring_deploying.rst

```diff
@@ -307,3 +307,117 @@ setting has been updated to no longer include the old node's public key, the
+
+If the Raft engine is peered properly, you should also see a group of messages
+that indicate if the node has been elected leader or if it has voted for another
+node as leader. On a leader node, the logs will look similar to this:
+
+.. code-block:: console
+
+  INFO  | raft::raft:833       | [12797589408118497989] became follower at term 0
+  INFO  | raft::raft:433       | [12797589408118497989] newRaft [peers: [9142281103993713288, 12797589408118497989], term: 0, commit: 0, applied: 0, last_index: 0, last_term: 0]
+  INFO  | raft::raft:833       | [12797589408118497989] became follower at term 1
+  INFO  | raft::raft:1120      | [12797589408118497989] is starting a new election at term 1
+  INFO  | raft::raft:848       | [12797589408118497989] became candidate at term 2
+  INFO  | raft::raft:950       | [12797589408118497989] received MsgRequestVoteResponse from 12797589408118497989 at term 2
+  INFO  | raft::raft:927       | [12797589408118497989] [logterm: 1, index: 2] sent MsgRequestVote request to 9142281103993713288 at term 2
+  INFO  | raft::raft:950       | [12797589408118497989] received MsgRequestVoteResponse from 9142281103993713288 at term 2
+  INFO  | raft::raft:1648      | [12797589408118497989] [quorum:2] has received 2 MsgRequestVoteResponse votes and 0 vote rejections
+  INFO  | raft::raft:891       | [12797589408118497989] became leader at term 2
+
+On a node that becomes a follower and votes for another node, the logs will look
+like this:
+
+.. code-block:: console
+
+  INFO  | raft::raft:833       | [9142281103993713288] became follower at term 0
+  INFO  | raft::raft:433       | [9142281103993713288] newRaft [peers: [9142281103993713288, 12797589408118497989], term: 0, commit: 0, applied: 0, last_index: 0, last_term: 0]
+  INFO  | raft::raft:833       | [9142281103993713288] became follower at term 1
+  INFO  | raft::raft:1014      | [9142281103993713288] [term: 1] received a MsgRequestVote message with higher term from 12797589408118497989 [term: 2]
+  INFO  | raft::raft:833       | [9142281103993713288] became follower at term 2
+  INFO  | raft::raft:1181      | [9142281103993713288] [logterm: 1, index: 2, vote: 0] cast MsgRequestVote for 12797589408118497989 [logterm: 1, index: 2] at term 2
+
+At this point, the Raft engine is running and ready to handle blocks.
+
+
+Troubleshooting and Common Issues
+=================================
+
+Below is a list of common issues that are seen when attempting to start and run
+a Sawtooth Raft network. If you encounter any problems, please consult this list
+to see if your issue is discussed. If you are still having problems, find out
+how to get help in the :doc:`/community/join_the_discussion` section.
```

Comment:
The link to the consensus channel is broken on the community page (should be `#sawtooth-consensus-dev`)